resources/usa-crafting/config.lua

```diff
@@ -822,6 +822,110 @@ Config.recipes = {
             type = "magazine",
             craftDurationSeconds = 1.5 * 60
         },
+        { 
+            name = "Hotwiring Kit",
+            image = "https://i.imgur.com/JZtZKoP.png",
+            requires = {
+                { name = "Fuse", quantity = 5 },
+                { name = "Plastic", quantity = 6 },
+                { name = "Electronic Kit", quantity = 4 }
+            },
+            produces = {
+                {
+                    name = "Hotwiring Kit",
+                    quantity = 1,
+                    type = "misc",
+                    weight = 5,
+                }
+            },
+            requiredCraftingLevel = 1,
+            type = "misc",
+            craftDurationSeconds = 3 * 60
+        },
+        { 
+            name = "Radio",
+            image = "https://i.imgur.com/FUreKPT.png",
+            requires = {
+                { name = "Metal Scraps", quantity = 3 },
+                { name = "Plastic", quantity = 6 },
+                { name = "Electronic Kit", quantity = 7 }
+            },
+            produces = {
+                {
+                    name = "Radio",
+                    quantity = 1,
+                    type = "misc",
+                    weight = 5,
+                }
+            },
+            requiredCraftingLevel = 1,
+            type = "misc",
+            craftDurationSeconds = 4 * 60
+        },
+        { 
+            name = "Sticky Bomb",
+            image = "https://i.imgur.com/gp0fW3J.png",
+            requires = {
+                { name = "Tape", quantity = 7 },
+                { name = "Plastic", quantity = 10 },
+                { name = "Electronic Kit", quantity = 5 },
+                { name = "Fuse", quantity = 3 },
+                { name = "Gun Powder", quantity = 5 }
+            },
+            produces = {
+                {
+                    name = "Sticky Bomb",
+                    quantity = 1,
+                    type = "misc",
```

Comment:
```suggestion
                    type = "weapon",
```
weapons need to have type = `weapon`

Reply:
makes me wonder if u tested this?

Reply:
i forgot to change this to weapon before i pushed it